### PR TITLE
Shippable move

### PR DESCRIPTION
**Pull request by notABluesSinger**
#### What are the links to relevant tickets?
https://hxshortbreaks.atlassian.net/browse/WEB-4809
#### What does this PR do?
Move the build tool to shippable
#### What functional/unit tests does this PR have?
Those that were left from the last developers work, but we now have full reporting on them.
#### How should a developer review this?
Reading.
#### Any background context you want to provide?
https://hxshortbreaks.atlassian.net/browse/FEA-442.  
The tests where commented out in a previous commit as they were failing. I found that they where not compatible with hapi v8, so I have adjusted them accordingly.
#### Screenshots (if appropriate)
Coverage
---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [ ] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @philmeehan
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "plugin-jsonapi",
   "description": "A hapi plugin for a jsonapi style output according to our 'house rules'",
-  "version": "0.1.1",
```
Comment (philmeehan) [inline package.json]: can't we bump to 1.0.0 now?

**Comment by robhuzzey** [comment]
I'll take reviewer 2

File: shippable.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - istanbul cover _mocha -- --recursive -R xunit > shippable/testresults/results.xml
+  - istanbul report cobertura --dir  shippable/codecoverage/
+
+# Setup our environment with the IAM credentials
+# we're using IAM user "shippable"
+env:
+  global:
+    - SLACK_CHANNEL=#ci-cd
+    # SLACK_WEBHOOK
+    - secure: hXVEupfO2c9Wnx+lGw0tvN0A83XymNgi6x9QaHvAN8xUyeqAKzsCXjP9nGbWtkNPpAucCbo3qZz6zMWKncsZKiM2rnuYQ2mU+m95n5CNvqoD2JWPXyX8Sz2kUVoFcZ20EB05KVRTv9IHqv0dWkdlvafLCR2HVZ3UJJhH0gjQBGJqNlhbJKS8j+n3JV20AoIPg4sy3LGJWoZgfpGVXlhXANgt9RKw3QfZibeHGNAR97Q+StLZJG6VoK1+jv9U78CY4RzsmU93LYuZCI5V6hOGR0CLo7jv1ggpvGffdpRig1XtQWEC4cIUhRlqurxYaRReZnHoFIdjTmnjBQg2eUnoqQ==
```
Comment (robhuzzey) [inline shippable.yml]: Should this be in our repo?

Comment (notABluesSinger) [inline shippable.yml]: Good question. This is an encrypted key, that can only be un-encrypted by shippable when ran via our account.

**Comment by robhuzzey** [comment]
+1 from me

**Pull request merged**